openshift/managed-velero-operator
# Add initial version of managed-velero-operator

Testing instructions:
- Has to be deployed to an actual AWS-based OpenShift cluster, as it requires the credential minter
- It will create an S3 bucket automatically. Make sure to clean it up (delete operator first, then delete bucket) before destroying your cluster.
- All the bits to install it are in the `deploy/` folder
- I have the latest operator image posted to `quay.io/cblecker/managed-velero-operator:latest` if you don't want to compile/host the image yourself.
